st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -164,8 +164,10 @@ def _post(self, payload, requester_user, show_secrets=False, match_multiple=Fals
             if action_alias_db.ack:
                 try:
                     if 'format' in action_alias_db.ack:
+                        message = render({'alias': action_alias_db.ack['format']}, result)['alias']
+
                         result.update({
-                            'message': render({'alias': action_alias_db.ack['format']}, result)['alias']
+                            'message': message
```

Comment:
Isn't this the same as `result['message'] = message` ?

Reply:
Yeah, it is - I just followed the pattern existing code uses.

I needed to refactor it because of the line length violation.